docker-compose.yml

```diff
@@ -25,9 +25,14 @@ services:
     restart: always
     logging: *default-logging
   vendor-management:
-    image: lblod/frontend-vendor-access-management:0.8.0
-    links:
-      - identifier:backend
+    # TODO replace with the actual released version
+    image: lblod/frontend-vendor-access-management:feature-acm-idm
```

Comment:
Once someone confirms the flow works I'll merge and release the frontend changes and bump it here.